Makefile.sparcv8leon3

```diff
@@ -0,0 +1,50 @@
+#
+# Makefile for Phoenix-RTOS 3
+#
+# SPARCv8 LEON3 options
+#
+# Copyright 2022 Phoenix Systems
+#
+# %LICENSE%
+#
+
+CROSS ?= sparc-phoenix-
+
+CC := $(CROSS)gcc
+CXX := $(CROSS)g++
+
+OLVL ?= -O2
+CFLAGS += $(OLVL)
+CFLAGS += -g -Wall -Wstrict-prototypes \
+	-mcpu=leon3 -mflat \
+	-DNOMMU \
+	-fdata-sections -ffunction-sections
+
+AR = $(CROSS)ar
+ARFLAGS = -r
+
+LD = $(CROSS)ld
+LDFLAGS := -z max-page-size=0x200 --gc-sections
+
+VADDR_KERNEL_INIT=40000000
+
+ifeq ($(KERNEL), 1)
+	STRIP = $(CROSS)strip
```

Comment:
Add `-ffixed-g6` to kernel `CFLAGS` to prohibit kernel from destroying pic base register during syscall perhaps? Or is there some other mechanism that save content of this register?

Reply:
Not needed afterall